sdk/monitor/monitor-opentelemetry/CHANGELOG.md

```diff
@@ -1,18 +1,21 @@
 # Release History
 
-## 1.1.2 (Unreleased)
+## 1.2.0 (2024-01-23)
```

Comment:
Looks like you have mixed changes including both releases, is this expected?

Reply:
Yes, we should be able to release both the distro & the exporter via the same PR. Looks like we did this last release as well https://github.com/Azure/azure-sdk-for-js/pull/27719.

Reply:
Ok just checking I though maybe you have another PR I just realized is the same as before